src/services/form-validation.ts

```diff
@@ -623,6 +639,15 @@ export function generateValidationSchema(
             submission,
             formElementsConditionallyShown,
           )
+
+          const repeatableSetExecutedLookups =
+            executedLookups !== undefined &&
+            typeof executedLookups !== 'boolean' &&
+            !Array.isArray(executedLookups) &&
```

Comment:
Not sure if this will work, but will likely be faster

```suggestion
             formElement.name in executedLookups &&
```

Reply:
Won't work without casting, which I didn't want to do